test/unit/processor.t.sol

```diff
@@ -238,4 +253,86 @@ contract VaultProcessUnitTest is Test, MainnetActors, Etches {
         vm.expectRevert();
         vault.processor(targets, values, data);
     }
+
+    function _setupValidatorRule() internal returns (MockValidator) {
+        bytes4 funcSig = bytes4(keccak256("deposit2()"));
+
+        IVault.ParamRule[] memory paramRules = new IVault.ParamRule[](2);
+
+        // First param rule for uint256 amount
```

Comment:
Param rules are never looked at when a validator is present, so we can just send an empty array here?

Reply:
yeah that is the case